sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSourceType.cs

```diff
@@ -5,22 +5,78 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    /// <summary>
+    /// The type of data source that ingests a <see cref="DataFeed"/> with data.
+    /// </summary>
     [CodeGenModel("DataSourceType")]
     public readonly partial struct DataFeedSourceType
     {
         /// <summary>
+        /// Azure Application Insights.
+        /// </summary>
+        public static DataFeedSourceType AzureApplicationInsights { get; } = new DataFeedSourceType(AzureApplicationInsightsValue);
+
+        /// <summary>
+        /// Azure Blob Storage.
+        /// </summary>
+        public static DataFeedSourceType AzureBlob { get; } = new DataFeedSourceType(AzureBlobValue);
+
+        /// <summary>
+        /// Azure Cosmos DB.
         /// </summary>
         [CodeGenMember("AzureCosmosDB")]
         public static DataFeedSourceType AzureCosmosDb { get; } = new DataFeedSourceType(AzureCosmosDbValue);
 
         /// <summary>
+        /// Azure Data Explorer.
+        /// </summary>
+        public static DataFeedSourceType AzureDataExplorer { get; } = new DataFeedSourceType(AzureDataExplorerValue);
+
+        /// <summary>
+        /// Azure Data Lake Storage Gen2.
```

Comment:
Do you think it is worth adding links for each service?

Reply:
Not sure. I would expect users to already have their data sources in place before even reaching this documentation, since you can't really start using Metrics Advisor without it.

Reply:
Great point. Ignore my comment then :)